content/en/docs/concepts/services-networking/service.md

```diff
@@ -716,6 +730,16 @@ the loadBalancer is set up with an ephemeral IP address. If you specify a `loadB
 but your cloud provider does not support the feature, the `loadbalancerIP` field that you
 set is ignored.
 
+To implement a Service of `type: LoadBalancer`, Kubernetes typically starts off
+by making the changes that are equivalent to you requesting a Service of
+`type: NodePort`, and then additionally configures the external load balancer to
```

Comment:
```suggestion
`type: NodePort`, and then additionally configuring the external load balancer to
```

Reply:
Hmm, I might reword this one. The suggestion you've made isn't idiomatic UK English - maybe I can find a wording that works for the US readers too.

Reply:
It just strikes me that you said "making the changes" and "configures the external load balancer". So you could instead say what I suggested or "makes the changes" and "configures the external load balancer". We probably need a grammar nerd to give us a ruling. :-)